apps/photos/src/services/download/index.ts

```diff
@@ -494,6 +543,15 @@ async function openThumbnailCache() {
     }
 }
 
+async function openFileCache() {
+    try {
+        return await CacheStorageService.open(CACHES.FILES, FILE_CACHE_LIMIT);
+    } catch (e) {
+        logError(e, 'Failed to open file cache');
+        return null;
```

Comment:
Swallow only for external users. Ideally we should know internally when it's failing